release/release_logs/2.0.0/microbenchmark.json

```diff
@@ -1,274 +1,274 @@
     ],
     "multi_client_put_gigabytes": [
-        34.625799191250174,
-        4.087652293153675
+        36.22342803548649,
+        1.9423599434343468
     ],
     "multi_client_tasks_async": [
-        35935.25738844591,
-        1631.6870881985358
+        34542.44693416779,
+        1451.966108026388
     ],
     "n_n_actor_calls_async": [
-        34777.026318884324,
-        294.2263921001007
+        34387.26578495381,
+        358.0603459980823
     ],
     "n_n_actor_calls_with_arg_async": [
-        2706.0679997684915,
-        28.564587555744374
+        2725.039943761716,
+        13.230389166349154
     ],
     "n_n_async_actor_calls_async": [
-        30029.38473720535,
-        291.9695966353376
+        28315.067181307353,
+        477.92349487311765
     ],
     "perf_metrics": [
         {
             "perf_metric_name": "single_client_get_calls_Plasma_Store",
             "perf_metric_type": "THROUGHPUT",
-            "perf_metric_value": 5402.762799881788
+            "perf_metric_value": 6270.806091875935
         },
         {
             "perf_metric_name": "single_client_put_calls_Plasma_Store",
             "perf_metric_type": "THROUGHPUT",
-            "perf_metric_value": 5389.520134405742
+            "perf_metric_value": 5382.000563319075
         },
         {
             "perf_metric_name": "multi_client_put_calls_Plasma_Store",
             "perf_metric_type": "THROUGHPUT",
-            "perf_metric_value": 10653.39172004253
+            "perf_metric_value": 11043.190408837896
         },
         {
             "perf_metric_name": "single_client_put_gigabytes",
             "perf_metric_type": "THROUGHPUT",
-            "perf_metric_value": 19.66716992775968
+            "perf_metric_value": 20.440651433770228
         },
         {
             "perf_metric_name": "single_client_tasks_and_get_batch",
             "perf_metric_type": "THROUGHPUT",
-            "perf_metric_value": 12.659666662711272
+            "perf_metric_value": 10.34283928313525
         },
         {
             "perf_metric_name": "multi_client_put_gigabytes",
             "perf_metric_type": "THROUGHPUT",
-            "perf_metric_value": 34.625799191250174
+            "perf_metric_value": 36.22342803548649
         },
         {
             "perf_metric_name": "single_client_get_object_containing_10k_refs",
             "perf_metric_type": "THROUGHPUT",
-            "perf_metric_value": 13.282060996229298
+            "perf_metric_value": 12.368070202442663
         },
         {
             "perf_metric_name": "single_client_tasks_sync",
             "perf_metric_type": "THROUGHPUT",
-            "perf_metric_value": 1424.3365828460285
+            "perf_metric_value": 1224.07938894957
         },
         {
             "perf_metric_name": "single_client_tasks_async",
             "perf_metric_type": "THROUGHPUT",
-            "perf_metric_value": 13149.808089478229
+            "perf_metric_value": 11258.337063584091
```

Comment:
feel  like we generally have 10% regression in single client cases. Do you know when this starts happening? Or is it just noise?